windows/vtpm_cold_add_remove/vtpm_add_remove_check.yml

```diff
@@ -0,0 +1,53 @@
+# Copyright 2023 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+- name: "Get keyId info in VM config"
+  include_tasks: ../../common/vm_get_cryptokey_id.yml
+
+# Check VM is encrypted after adding vTPM device
+- name: "Make sure keyId is set in VM config"
+  ansible.builtin.assert:
+    that:
+      - vm_cryptokey_id_info
+      - vm_cryptokey_id_info.keyId
+      - vm_cryptokey_id_info.providerId.id
```

Comment:
Here needs to check `vm_cryptokey_id_info.keyId is defined` and `vm_cryptokey_id_info.providerId.id is defined` before checking their value.